grass7/raster/r.null.all/r.null.all.py

```diff
@@ -0,0 +1,190 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+############################################################################
+#
+# MODULE:       r.null.all
+# AUTHOR(S):    Vaclav Petras <wenzeslaus gmail com>
+#
+# PURPOSE:      Manage null values for all raster maps in a mapset
+# COPYRIGHT:    (C) 2019 by Vaclav Petras and the GRASS Development Team
+#
+#               This program is free software under the GNU General
+#               Public License (>=v2). Read the file COPYING that
+#               comes with GRASS for details.
+#
+#############################################################################
+
+#%module
+#% description: Manages NULL values of raster maps in a mapset or their subset
+#% keyword: raster
+#% keyword: map management
```

Comment:
Probably the order of keyworks should follow that of r.null (?) or both be aligned.

Reply:
r.null has only `raster, null data`, so I think we need to fix it.

The questions here are more general: 1) Should be we have synonyms in keywords, i.e., both `null data` and `no-data` 2) Should we repeat the words from the name of module, i.e., `null` or `null data`? 3) Should different spellings be included, e.g., `no-data` and `NoData`?   I'm saying yes first two questions, so that's why I included `raster, map management, null data, no-data`. What do you think?